### PR TITLE
catgirl: 1.9 -> 1.9a

### PR DESCRIPTION
**Pull request by wahjava**
###### Motivation for this change

As I use it, and noticed there is an update

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sternenseemann** [comment]
You'll need to rebase your branch on `master` and remove the patch added in 711b572d22a1f2dd285a86feb85781334b572948 since it is included in the 1.9a release.

**Comment by wahjava** [comment]
> You'll need to rebase your branch on `master` and remove the patch added in [711b572](https://github.com/NixOS/nixpkgs/commit/711b572d22a1f2dd285a86feb85781334b572948) since it is included in the 1.9a release.

Thanks for pointing it out.

File: pkgs/applications/networking/irc/catgirl/default.nix [hunk 1]
```diff
@@ -2,20 +2,13 @@
...
+  version = "1.9a";
 
   src = fetchurl {
     url = "https://git.causal.agency/catgirl/snapshot/${pname}-${version}.tar.gz";
-    sha256 = "182l7yryqm1ffxqgz3i4lcnzwzpbpm2qvadddmj0xc8dh8513s0w";
+    sha256 = "sha256-MEm5mrrWfNp+mBHFjGSOGvvfvBJ+Ho/K+mPUxzJDkV0=";
   };
 
-  patches = [
-    (fetchpatch {
```
Comment (sternenseemann) [inline pkgs/applications/networking/irc/catgirl/default.nix:13]: Can you also remove the now unused `fetchpatch` input while we're at it?

**Review by sternenseemann — APPROVED** [review]
Other than that LGTM, seems to work.

**Comment by risicle** [comment]
Builds & runs, macos 10.15 & nixos x86_64

**Pull request merged**